README.md

```diff
@@ -147,9 +147,9 @@ public class FooWidget {
         this(data, Optional.empty());
```

Comment:
This needs to be changed to `this(data, null)`, or you need to keep both `Foo(String, Optional<Bar>)` and `Foo(String, Bar)` constructors.